pkg/distributor/distributor.go

```diff
@@ -524,7 +524,7 @@ func (d *Distributor) forAllIngesters(f func(client.IngesterClient) (interface{}
 }
 
 // LabelValuesForLabelName returns all of the label values that are associated with a given label name.
-func (d *Distributor) LabelValuesForLabelName(ctx context.Context, labelName model.LabelName) (model.LabelValues, error) {
+func (d *Distributor) LabelValuesForLabelName(ctx context.Context, labelName model.LabelName) ([]string, error) {
```

Comment:
Why the switch to string instead of model.LabelValue?

Reply:
It just saved a few conversions; generally upstream is moving away from using these types anyway.

Reply:
ref #765